docs/personalization/api_reference/tracking_api.md

```diff
@@ -387,7 +373,7 @@ See [Recommendation API](recommendation_api.md) for more details.
 A trigger message includes requests for a Triggeropened and Clicktriggered event.
```

Comment:
Maybe use the properties of the trigger message below, `triggerOpenedLink` and `clickTriggered`?